modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -0,0 +1,103 @@
+= Advanced Template plugin
+:navtitle: Advanced Template
+:description: Create and edit complex templates in {productname} for use with {productname}.
+:description_short: Create templates.
+:keywords: plugin, advanced template, templates, advtemplate.
+:pluginname: Advanced Template
+:plugincode: advtemplate
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
+The Advanced Template Premium plugin enables the creation of complex, interactive templates using {productname}.
```

Comment:
```suggestion
The {pluginname} Premium plugin enables the creation of complex, interactive templates using {productname}.
```
Might be a good idea to grep this, in case we change the plugin name.